src/shrdlu/loader.19

```diff
@@ -49,81 +70,100 @@
 
 (SETQ VERSION-FILES NIL) 
 
+(defun loadplanner ()
+       (ALLOC '(LIST 320000
+		FIXNUM 15000
+		SYMBOL 15000
+		array 500
+		flonum 4000))
+       (SETQ PURE NIL)
+       (setq car t)
+       (setq cdr t)
+       (SETQ THINF NIL THTREE NIL THLEVEL NIL)
+       (MAPC 'LOADER '(PLNR THTRAC))
+       (THINIT))
+
 (DEFUN LOADSHRDLU NIL 
        (ALLOC '(LIST 320000
 		     FIXNUM
 		     15000
 		     SYMBOL
 		      15000
-                     array 500 ))
+                     array 500
+		     flonum 3000))
        (SETQ PURE NIL)
+       (setq car t)
+       (setq cdr t)
+       (SETQ THINF NIL THTREE NIL THLEVEL NIL)
+       (load '((lisp) slave fasl))
        (MAPC 'LOADER '(PLNR THTRAC))
        (THINIT)
-       (SETQ THINF NIL THTREE NIL THLEVEL NIL)
        (setq errlist nil)   ;removes micro-planner's fangs
        (MAPC 'LOADER '(SYSCOM MORPHO SHOW))
        (MAPC 'LOADER '(PROGMR GINTER GRAMAR DICTIO))
        (MAPC 'LOADER '(SMSPEC SMASS SMUTIL))
        (LOADER 'NEWANS)
-       (MAPC 'LOADER '(BLOCKS DATA))
-       (FASLOAD TRACE FASL COM COM)
-       (FASLOAD GRAPHF FASL DSK SHRDL1)
+       (load 'blockp)
+       (load 'blockl)
+       (load 'data)
+       (load '((lisp) trace fasl))
+       (load '((shrdlu) graphf fasl))
        (LOADER 'SETUP)
-       (fasload grindef fasl com com)
+       (load '((lisp) grinde fasl))
        'CONSTRUCTION/ COMPLETED) 
 
 (defun loadparser nil
 	(mapc 'loader '(syscom morpho show))
 	(mapc 'loader '(progmr ginter gramar dictio))
 	(loader 'setup)
-	(load parser > dsk shrdl1)		;load is defined on ddm;*load >
+	(load parser > dsk xshrdl)		;load is defined on ddm;*load >
```

Comment:
Curious what this xshrdl is?

Reply:
It should be SHRDLU. At one point, I had multiple SHRDLU directories (for testing things) and I was loading from XSHRDL. I thought I had changed them all back (was using TECO to edit from an iPhone TELNET connection where GUI was all messed up; and I must have missed some.